src/network/MissionControlProtocol.cpp

```diff
@@ -56,7 +57,6 @@ void MissionControlProtocol::handleEmergencyStopRequest(const json& j) {
 		_power_repeat_task.start();
```

Comment:
Are we properly handling this case in which we leave e-stop?

Reply:
Technically we can't leave e-stop, we're supposed to restart the rover code when we e-stop. (hence why there's no undo method for `robot::emergencyStop()`, so we should probably factor that into the logic here, but I think that's out of the scope of this PR, and should be done in the same PR that enforces the e-stop flag in the world interface.

Reply:
Gotcha! Makes sense.